opensrp-chw-hf/src/main/java/org/smartregister/chw/hf/interactor/LDVisitInteractor.java

```diff
@@ -80,6 +81,8 @@ public void calculateActions(BaseLDVisitContract.View view, MemberObject memberO
                     evaluateHBTest(details);
                 }
 
+                evaluateSyphilisTest(details);
```

Comment:
This needs a check of whether the client was not tested for Syphilis before during L&D Registration. 
i.e `test_not_conducted` in syphilis during `labour_and_delivery_registration_anc_clinic_findings.json`

Reply:
ooh... I did not see that requirement, let me add that